modules/imgimport/include/imgimport.h

```diff
@@ -35,6 +35,7 @@
 
 #include "frame.h"
 #include <opencv2/highgui/highgui.hpp>
+#include <DeckLinkAPI.h>
```

Comment:
I don't think the ImageImport header should be including DeckLink, It's not currently using it and shouldn't be since the decklink stuff should only be used on the implementation side.